modules/ROOT/pages/7.7.0-release-notes.adoc

```diff
@@ -169,6 +169,13 @@ In previous versions of {productname}, hovering over toolbar menu item would dis
 
 {productname} {release-version} resolves this issue by replacing the `title` attribute with the `aria-label` attribute for toolbar groupings, preventing the default browser tooltip from appearing.
 
+=== Fixed keyboard navigation for size inputs in context forms.
+// #TINY-11394
+
+Previously, input and slider elements in the context toolbar did not support keyboard navigation, preventing users from accessing these elements via the keyboard.
```

Comment:
now is also possible to "manipulate" the slider using the arrow keys (left and right), I don't know if this should be made explicit or if it is implicit in your sentence (sorry English is not my first language :sweat_smile: )

Reply:
```suggestion
Previously, input and slider elements in the context toolbar did not support keyboard navigation, preventing users from accessing and adjusting these elements via the keyboard.
```